src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -13,6 +13,22 @@
                     #{studyId}
                 </foreach>
             </if>
+            <!-- filter for samples which belong to the sample lists (aka case lists) selected by the user and appearing in the studyViewFilter.caseLists collection -->
+            <if test="studyViewFilter.caseLists != null and !studyViewFilter.caseLists.isEmpty()">
+                INTERSECT
+                SELECT sample_unique_id
+                FROM sample_list_list sll
+                    LEFT JOIN sample_view sv ON sll.sample_id=sv.internal_id
+                    LEFT JOIN sample_list sl on sll.list_id=sl.list_id
```

Comment:
Same as below I believe this could use the sample_list_columnstore table to do this